### DISCUSSION TITLE
Standup (Wednesday, November 16th)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [BBGTC/food-bank-app#64] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Sent PR regarding connection with back-end

**What are your priorities for the next 24 hours?**:
 - Work on iOS homework

**Blockers**:
 - Homework from other assignments

**Shoutouts**:
 - @DavBot02  for turning to the dark side

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #53 
 - Fixed some problems with package versions

**What are your priorities for the next 24 hours?**:
 - Check qr content

**Blockers**:
 - Fedora is weird

**Shoutouts**:
 - @antonieto i love him

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished field validation #60 and solved some merge confl

**What are your priorities for the next 24 hours?**:
 - turn in ken bauer's homework
 - start the work on the client application for veryfing donations

**Blockers**:
 - I had been working on the algorithms assignment (finished two of four algorithm implementations).

**Shoutouts**:
 - @CarlosCRG19 for the nice hook
